ecommerce/monitoring/grafana/provisioning/dashboards/dashboards.yml

```
@@ -0,0 +1,13 @@
+apiVersion: 1
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)